commands/quotes.py

```diff
@@ -57,6 +57,27 @@ async def get_quotes(ctx: discord.message, client: discord.client):
         await ctx.channel.send(embed = create("GetQuotes Reply", "Invalid Syntax! You need integers", "red"))
 
 
+async def remove_quote(ctx: discord.message, client: discord.client):
+    if not (author_has_role(ctx, "admin") or author_has_role(ctx, "admins()")):
```

Comment:
Note to self: the function call to check if the user has "Admin" or "Admins()" should be reflected in the lower level function to check role in something like author_has_admin_perms()